### PR TITLE
Make all NDP configuration parameters user configurable

### PR DESCRIPTION
**Pull request by laurentderu**
Trivial PR allowing user configuration of all the NDP parameters related to Router Advertisement.

### CODE REVIEW

File: core/net/ipv6/uip-nd6.h [hunk 1]
```diff
@@ -60,11 +60,23 @@
 /** \name RFC 4861 Host constant */
 /** @{ */
 /** \brief Maximum router solicitation delay */
+#ifndef UIP_ND6_MAX_RTR_SOLICITATION_DELAY
```
Comment (nvt) [inline core/net/ipv6/uip-nd6.h]: Shouldn't you check for UIP_CONF_ND6_MAX_RTR_SOLICITATION_DELAY? Similarly for UIP_CONF_ND6_RTR_SOLICITATION_INTERVAL below.

Comment (laurentderu) [inline core/net/ipv6/uip-nd6.h]: Yes, I was too fast and I overlooked that...

**Comment by nvt** [comment]
:+1:

**Comment by greg-king5** [comment]
I'm sorry that I got to this PR five days too late.

The new configuration names are spelled wrong.  They should start with the name of the module: "uip-nd6".  Example: `UIP_ND6_CONF_MAX_RTR_SOLICITATION_DELAY`

**Pull request merged**